docs/src/appendix/snfoundry-toml.md

```diff
@@ -0,0 +1,103 @@
+# The Manifest Format
+
+The `snfoundry.toml` file, present in each package, is called its manifest. 
+This file allows you to specify sncast settings. It is written in the [TOML](https://toml.io/) format. 
+It contains metadata needed to configure sncast settings and should be placed in the root of your project. 
+If `snfoundry.toml` is not found in the root directory, sncast will look in all parent directories. 
+If it is not found, default values will be used.
+
+Schema of the `snfoundry.toml` file:
+
+## `[sncast.<profile-name>]`
+
+
+```toml
+[sncast.myprofile]
+url = "http://127.0.0.1:5055/rpc"
+account = "user1"
+accounts_file = "../account-file"
+wait_params = { timeout = 300, retry_interval = 5 }
+```
+
+There are no required fields in the profile section. It's up to user which arguments do they want to provide in advance and which specify via CLI.
+
+### `url`
+
+The `url` field specifies the address of RPC provider.
+
+```toml
+[sncast.myprofile]
+url = "http://example.com"
+```
+
+### `accounts_file`
+
+The `accounts_file` field specifies the path to a file containing account information. 
+If not provided, the default path is `~/.starknet_accounts/starknet_open_zeppelin_accounts.json`.
+
+```toml
+[sncast.myprofile]
+accounts_file = "path/to/accounts.json"
+```
+
+### `account`
+
+The `account` field specifies which account from the `accounts_file` to use for transactions.
+
+```toml
+[sncast.myprofile]
+account = "user-dev"
+```
+
+### `keystore`
+
+The `keystore` field specifies the path to the keystore file.
+
+```toml
+[sncast.myprofile]
+keystore = "path/to/keystore"
+```
+
+### `wait_params`
+
+The `wait_params` field specifies the parameters for waiting during transactions. Default values are `timeout = 300` and `retry_interval = 5`.
+
+```toml
+[sncast.myprofile]
+wait_params = { timeout = 300, retry-interval = 5 }
+```
+
+### `block_explorer`
+
+The `block_explorer` field specifies the block explorer service used to display links to transaction details. 
+The `Sepolia Testnet` is supported only by `Voyager` and `StarkScan` block explorers.
+
+| Value     | URL                                    |
+|-----------|----------------------------------------|
+| StarkScan | `https://starkscan.co`          |
+| Voyager   | `https://voyager.online`               |
+| ViewBlock | `https://viewblock.io/starknet`        |
+| OkLink    | `https://www.oklink.com/starknet`      |
+| NftScan   | `https://starknet.nftscan.com`         |
+
+```toml
+[sncast.myprofile]
+block_explorer = "StarkScan"
+```
+
```

Comment:
This is probably duplicating information we already have in a relevant section.

Reply:
There is no other place in docs, where supported block explorers are listed.

Reply:
Okay, that's not great. Let's keep your version for now, but eventually this should be documented in a more relevant place.

Reply:
#2487